### PR TITLE
Add some clarification regarding docs.count

### LINKED ISSUE #17319: ES 2.2.1 strange document count in _cat/indices
**Elasticsearch version**: 2.2.1
**JVM version**: Oracle Java 8 (1.8.0_74)
**OS version**: Ubuntu Server 14.04.4 LTS
**Description of the problem including expected versus actual behavior**: `/_cat/indices` returns twice number of docs (`docs.count` column) than exists in the index.

```
curl -XGET 'http://127.0.0.1:9200/_cat/indices?v'
health status index                   pri rep docs.count docs.deleted store.size pri.store.size
green  open   index-1doc              2   1          2            0     39.9kb         19.9kb
green  open   index-1doc-1s0r         1   0          2            0     19.8kb         19.8kb
green  open   index-10000doc-7s3r     7   3      20000            0     76.3mb           19mb
```

**Steps to reproduce**:
 1. Feed a new index with N documents using bulk insert
 2. Query `/_cat/indices` and check `docs.count` column

### PR DESCRIPTION
**Pull request by ghost**
Closes #17319

### CODE REVIEW

**Comment by jasontedor** [comment]
@hgfischer Thanks, can you sign the CLA?

File: docs/reference/cat/indices.asciidoc [hunk 1]
```diff
@@ -13,8 +13,9 @@ green  open   twitter2   2   0       2030            0      5.8mb          5.8mb
 --------------------------------------------------
 
 We can tell quickly how many shards make up an index, the number of
-docs, deleted docs, primary store size, and total store size (all
-shards including replicas).
+docs at Lucene level (eg.: including nested documents), deleted docs,
```
Comment (jasontedor) [inline docs/reference/cat/indices.asciidoc]: Can we change this to say "at the Lucene level (including hidden docs from e.g., nested docs)"?

File: docs/reference/cat/segments.asciidoc [hunk 2]
```diff
@@ -44,6 +44,8 @@ generation::    The generation number is incremented with each segment that is w
                 The name of the segment is derived from this generation number.
 
 docs.count::    The number of non-deleted documents that are stored in this segment.
+                Note that these are Lucene documents, so the count will include root and
```
Comment (jasontedor) [inline docs/reference/cat/segments.asciidoc]: Can we change this to say "will include hidden documents (e.g., from nested types)?"

**Comment by hgfischer** [comment]
@jasontedor I already signed the CLA, two times.

**Comment by clintongormley** [comment]
@hgfischer the problem with the CLA was using a different email address from github.  I've connected the two now.

**Comment by hgfischer** [comment]
@clintongormley Strange. I've signed with the same address from my profile. Nevertheless, this PR still show that I didn't signed.

**Comment by clintongormley** [comment]
@hgfischer you signed with your gmail address, but here you used your .io address.  Anyway, CLA database updated to include both.

**Comment by hgfischer** [comment]
Ah, ok. It was in my `.gitconfig` file. Good to know. Thanks

**Comment by clintongormley** [comment]
@hgfischer will merge this in as soon as you've pushed a commit to address @jasontedor 's comments

File: docs/reference/cat/indices.asciidoc (hunk 1 above)
Comment (hgfischer) [inline docs/reference/cat/indices.asciidoc]: Sure.

**Comment by hgfischer** [comment]
@clintongormley done.

**Comment by clintongormley** [comment]
thanks @hgfischer - merged

**Comment by hgfischer** [comment]
Great! :)

**Pull request merged**